test/emqx_connection_SUITE.erl

```diff
@@ -36,6 +36,8 @@ init_per_suite(Config) ->
     ok = meck:new(emqx_channel, [passthrough, no_history, no_link]),
     %% Meck Cm
     ok = meck:new(emqx_cm, [passthrough, no_history, no_link]),
+    ok = meck:expect(emqx_cm, increment_connected_client_count, fun() -> ok end),
```

Comment:
do we still need to stub this call ?

Reply:
I believe that we do, since I added those calls to `emqx_channel:ensure_{connected,disconnected}`, these tests that mock `emqx_cm` fail because they do not start up an `emqx_cm` and its tables.